### ISSUE TITLE
Typo Preventing html_attrs defaults from working

### ISSUE THREAD

**Issue by zachbellay** [EmilStenstrom/django-components#512]
Hey @EmilStenstrom @JuroOravec, I have been trying to get the following code to work:

#### `list.html`
```html
{% component "card" attrs:@click="console.log('hello world')" %}
{% endcomponent %}
```

#### `card.py`
```py
from django_components import component

@component.register("card")
class Card(component.Component):
    template_name = "card/template.html"

    def get_context_data(self, attrs:dict=None, **kwargs):
        if not attrs:
            attrs = {}
        context_data = super().get_context_data(**kwargs)
        context_data.update(kwargs | {'attrs':attrs})
        return context_data
```

#### `card/template.html`
```html
<div {% html_attrs attrs defaults:class="p-4 bg-white border-2 border border-gray-200 rounded-lg shadowdark:border-gray-600 my-4" %}>
    {% slot "content" default %}
    {% endslot %}
</div>
```

The current output is:

```html
<div @click="console.log('hello world')">
</div>
```

but what I expect is:

```html
<div class="p-4 bg-white border-2 border border-gray-200 rounded-lg shadowdark:border-gray-600 my-4" @click="console.log('hello world')">
</div>
```

Maybe I have my mental model wrong, and if that's the case please let me know.

But I believe the cause of this issue is this following typo:

https://github.com/EmilStenstrom/django-components/blob/dd292b03508b7c03b2ad4f9fb9c17dec747845ba/src/django_components/attributes.py#L58

```diff
- default_attrs = attrs_and_defaults_from_kwargs.get("defualts", {})
+ default_attrs = attrs_and_defaults_from_kwargs.get("defaults", {})
```

**Comment by zachbellay** [comment]
Opened a tiny PR for convenience: https://github.com/EmilStenstrom/django-components/issues/512

**Comment by JuroOravec** [comment]
Thanks @zachbellay! I've updated the tests, merged, and the fix has now been released in v0.78 🎉
